roles/wildfly_systemd/meta/argument_specs.yml

```diff
@@ -127,6 +127,26 @@ argument_specs:
                 default: "{{ wildfly_home }}"
                 description: "Install base directory for instance, ie. for colocated installations"
                 type: "str"
+            wildfly_systemd_port:
+                default: "9990"
+                description: "Main port where wildfly instance will be running"
```

Comment:
Please describe it as "The port to wait for when starting up wildfly if wildfly_systemd_wait_for_port is true"